ui_testing/pages/contact_page.py

```diff
@@ -256,8 +256,9 @@ def create_update_contact(self, create=True, no='', name='', address='', postalc
         else:
             contact_data['contact_persons'] = []
 
-        self.base_selenium.LOGGER.info('Saving the contact created')
-        self.save(save_btn='contact:save')
+        if save:
+            self.base_selenium.LOGGER.info('Saving the contact created')
+            self.save(save_btn='contact:save')
         return contact_data
```

Comment:
if save=False, it will return contact data also

Reply:
Yes it will return the data